assets/cms/components/IconButton.vue

```diff
@@ -0,0 +1,70 @@
+<template functional>
+    <span class="button-container">
+        <button
+            @click='listeners.click'
+            v-bind:class="['btn', props.bootstrapClass]"
+            :disabled="props.disabled"
+            >
+            <span class="label" v-if="props.labelPosition === 'left'"
+                v-bind:class="{disabled : props.disabled}">
+                <slot />
+            </span>
+
+            <Icon :icon="props.icon" :type="props.type" :color="props.disabled ? '#ccc' : props.color"/>
+
+            <span class="label" v-if="props.labelPosition === 'right' || !props.labelPosition"
+                v-bind:class="{disabled : props.disabled}">
+                <slot />
+            </span>
+
+        </button>
+    </span>
+</template>
+
+<script>
+import Icon from './Icon'
+
+export default {
+    props: {
+        outline: Boolean,
+        disabled: Boolean,
+        labelPosition: String,
+        icon: String,
+        type: String,
+        color: String,
+        bootstrapClass: {
```

Comment:
Instead of just requesting the bootstrap class we should define types for these buttons, see the `format` prop in the task